lib/gooddata/lcm/actions/synchronize_attribute_drillpaths.rb

```diff
@@ -45,7 +45,8 @@ def call(params)
               client.projects(pid) || fail("Invalid 'to' project specified - '#{pid}'")
             end
 
-            GoodData::LCM.transfer_attribute_drillpaths(from_project, to_projects)
+            attributes = from_project.attributes
+            GoodData::LCM.transfer_attribute_drillpaths(from_project, to_projects, attributes)
```

Comment:
Could you tell me why we can enhance the performance by this solution? AFAIK, we only have ONE source project and MANY target projects, so caching the source attributes here is the same as what we've done in `GoodData::LCM.transfer_attribute_drillpaths`, that means we enhance nothing.